tools/alpine/Makefile

```diff
@@ -9,25 +9,25 @@ default: push
 show-tag:
 	@sed -n -e '1s/# \(.*\/.*:[0-9a-f]\{40\}\)/\1/p;q' versions
 
-hash: Dockerfile Makefile packages
+iid: Dockerfile Makefile packages
```

Comment:
You mean rebase this without the first commit? Can do.

(sorry for the ordering, GH dynamic update seems a bit flakey for me at the moment so I only saw your first comment at first)